modin/core/dataframe/pandas/partitioning/partition_manager.py

```diff
@@ -137,7 +134,9 @@ def column_partitions(cls, partitions):
         if not isinstance(partitions, list):
             partitions = [partitions]
         return [
-            cls._column_partitions_class(col) for frame in partitions for col in frame.T
+            cls._axis_partitions_class(col, axis=1)
```

Comment:
The main change is passing the `axis` parameter to the constructor. In other words - now `axis` is an attribute of an object, not a class.